src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "../eltwise.hpp"
+#include "arm_compute/runtime/NEON/NEFunctions.h"
+
+namespace ov {
+namespace intel_cpu {
+
+arm_compute::TensorShape eltwiseShapeCast(const VectorDims& dims);
+
+class AclEltwiseExecutor : public EltwiseExecutor {
+public:
+    AclEltwiseExecutor(const ExecutorContext::CPtr context);
+
+    bool init(const EltwiseAttrs& eltwiseAttrs,
+              const std::vector<MemoryDescPtr>& srcDescs,
+              const std::vector<MemoryDescPtr>& dstDescs,
+              const std::vector<EltwisePostOp>& postOps) override;
+
+    void exec(const std::vector<MemoryCPtr>& src,
+              const std::vector<MemoryPtr>& dst,
+              const void *post_ops_data_) override;
+
+    impl_desc_type getImplType() const override {
+        return implType;
+    }
+private:
+    EltwiseAttrs aclEltwiseAttrs{};
+    impl_desc_type implType = impl_desc_type::acl;
+    arm_compute::Tensor src1Tensor;
+    arm_compute::Tensor src2Tensor;
+    arm_compute::Tensor dstTensor;
+    std::function<void()> exec_func;
+    bool is_unary_op = false;
+};
+
+class AclEltwiseExecutorBuilder : public EltwiseExecutorBuilder {
+public:
+    bool isSupported(const EltwiseAttrs& eltwiseAttrs,
+                     const std::vector<MemoryDescPtr>& srcDescs,
+                     const std::vector<MemoryDescPtr>& dstDescs) const override {
+        for (const auto &desc : srcDescs) {
+            if (desc->hasLayoutType(LayoutType::nCsp8c) ||
+                desc->hasLayoutType(LayoutType::nCsp16c)) {
```

Comment:
In case of introduction new type inside LayoutType this condition becomes invalid for ACL Eltwise. It is always better to directly list **supported** types

Reply:
done